### DISCUSSION TITLE
How to deal with additional file(s) required by a geo file

### DISCUSSION THREAD

**Discussion by Trophime** [feelpp/feelpp#1651] [Q&A]
Hi,
I have a geo file that  is build using `Merge` gmsh command.

Merge "HR-54-251_sans_micro-canaux.mesh";

How do I tell feelpp to look for this file in the cfg?

**Comment by Trophime** [comment]
@vincent any idea how to do that? I think that we would need this feature for testing Abdoulye work

**Reply by vincentchabannes** [reply]
We copy the geofile before run the meshing, I think currently there is no solution.
But you can simply generate the .msh from this geo.

**Reply by Trophime** [reply]
For adaptive mesh, it will be great to be able to have the files defined by Include and/or Merge in geo files.
Using for instance: Include "CAD.brep", we could certainly remesh the geometry starting from CAD file instead of using the previous mesh. As a consequence, we would have better control on boundaries. Cecile did something like that in the early days of Hifimagnet
